source/core/backups.txt

```diff
@@ -17,15 +17,25 @@ events.
 Back Up with Atlas
 ------------------
 
-.. note::
+MongoDB Atlas, the official MongoDB cloud service, provides 2 fully-managed
+methods for backups:
+
+1. :atlas:`Continuous Backups </backup/continuous-backup>`, which take
+   incremental backups of data in your cluster, ensuring your backups are
+   typically just a few seconds behind the operational system. Atlas continuous
+   backups allow you to restore from stored snapshots or from a selected point
+   in time within the last 24 hours.  You can also query a continuous backup
+   snapshot.
 
-   If your database is hosted on MongoDB Atlas, you don’t need to manually back
-   up and restore your data. Instead, you can use the fully-managed, continuous
-   backup service to take incremental snapshots of your data and restore to any
-   point-in-time.
+#. :atlas:`Cloud Provider Snapshots </backup/cloud-provider-snapshots>`, which
+   provide localized backup storage using the native snapshot functionality of
+   the cluster's cloud service provider.
+
+.. note::
 
-   Not yet hosting your database on Atlas? `Get started now
-   <https://www.mongodb.com/cloud/atlas?utm_source=backups&utm_campaign=20-docs-in-20-days&utm_medium=docs>`_.
+   If you are not currently using MongoDB Atlas, can learn how to migrate your
```

Comment:
@kirbyk  -- did you want ", you can learn" or ", learn"  instead of ",  can learn"